assets/panel/debugger.properties

```diff
@@ -392,6 +392,10 @@ callStack.collapse=Collapse rows
 # message to show more of the frames.
 callStack.expand=Expand rows
 
+# LOCALIZATION NOTE (components.header): Header for the React Components pane 
```

Comment:
this should be `Header for the Framwork Components`

we want to add Ember soon :)